traitsui/testing/tester/qt4/implementation/list_editor.py

```diff
@@ -0,0 +1,121 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+from traitsui.testing.tester import locator
+
+from traitsui.qt4.list_editor import (
+    CustomEditor,
+)
+
+
+def find_by_name_in_nested_ui(wrapper, location):
+    """ Helper function for resolving from an _IndexedCustomEditor to a
+    TargetByName.
+
+    Parameters
+    ----------
+    wrapper : UIWrapper
+    location : instance of locator.TargetByName
+    """
+    new_interactor = wrapper.locate(locator.NestedUI())
+    return new_interactor.find_by_name(location.name).target
+
+
+class _IndexedCustomEditor:
```

Comment:
I have been thinking about how to modify this so that potentially repeated code doesn't need to be repeated.  For example trying to extract some skeleton of this class and keep it in the common_ui_targets module, but I could not decide on a clean way to do so.  The specifics of accessing the nested UI will vary from editor to editor

Reply:
Sounds reasonable to me!